### PR TITLE
Mergeup from master to stable

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

File: configs/components/leatherman.json [hunk 1]
```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/leatherman.git", "ref": "0.8.1"}
```
Comment (McdonaldSeanp) [inline configs/components/leatherman.json]: ping @MikaelSmith on this change, do we need leatherman 0.9.0 for ruby 2.3? If not I would like to keep leatherman 0.8.1 for the next release.

Comment (MikaelSmith) [inline configs/components/leatherman.json]: 0.9.0 is necessary if we're building a newer version of Facter. If we're sticking to facter#stable, we can probably stay at 0.8.1.

Comment (McdonaldSeanp) [inline configs/components/leatherman.json]: Thanks

**Comment by geoffnichols** [comment]
👍

**Pull request merged**